requirements/quality.txt

```diff
@@ -32,13 +32,13 @@ urllib3==1.25.9           # via requests
 # $ pip install pylint edx-lint
 # $ pip freeze > new.txt
 # And replace the contents of this file with the lines in new.txt that don't exist in old.txt
-astroid==1.4.9
-configparser==3.5.0
-edx-lint==0.5.2
-lazy-object-proxy==1.2.2
+astroid
```

Comment:
I think that this could cause problems in the future  (Have unpinned versions in this file). Maybe it would be better to remove the requirements/quality-extra.in file, and just add edx-lint to the quality.in file